templates/html/.partials/topics.html

```diff
@@ -1,4 +1,6 @@
+{{#if asyncapi.topics}}
```

Comment:
I don't think this is needed. We're not including the partial in the other change, so it's safe.

Reply:
I've checked. It does appear without the condition...

Reply:
Yes, because this is the place where you should place the `if` condition: https://github.com/asyncapi/generator/blob/master/templates/html/.partials/content.html#L3

Reply:
The one on the index.html is the sidebar that only appears when you download the documentation.

Reply:
I moved the condition to the content.html.
Regarding the sidebar: Well, Im using the generator locally. Generating an html, which will be hosted in an intranet... and, the sidebar has the Topics block. Thats why I made it conditional too.

Reply:
Yeah, that's good to me. Thanks for taking the time!

Reply:
Haha! Well, thanks for this repo and the clarification over Sclack too! ; )